eval.c

```diff
@@ -43,7 +43,7 @@ extern ID ruby_static_id_cause;
      (BUILTIN_TYPE(obj) == T_CLASS || BUILTIN_TYPE(obj) == T_MODULE))
 
 /*!
- * Initializes the Ruby VM and builtin libraries.
+ * Initializes the VM and builtin libraries.
```

Comment:
I dropped `Ruby` because it's clear from the function name, and just `VM` is used in other cases as well such as `ruby_finalize` and `ruby_cleanup`.